manubot/process/manuscript.py

```diff
@@ -3,24 +3,6 @@
 import json
 import logging
 import pathlib
-import re
-
-from manubot.cite.citekey import citekey_pattern, is_valid_citekey
-
-
-def get_citekeys(text):
```

Comment:
I'm not reviewing the removed functions carefully.  The test cases should be sufficient.